src/config/dynamic-theme-fixes.config

```diff
@@ -16010,6 +16010,8 @@ div.post-content.footer-content > h2 > img
 .mw-hiero-outer.mw-hiero-table
 #p-logo-text
 body > .oo-ui-windowManager .vega .marks
+.minerva-footer-logo img
+.music-symbol
```

Comment:
Where can I find these two elements?

Reply:
The first is at the bottom of Wikipedia [mobile page](https://en.m.wikipedia.org/wiki/Main_Page).

The second is at https://en.wikipedia.org/wiki/Tempo#Tempo%E2%80%93rhythm_interaction